### ISSUE TITLE
Problem with uploading on MIUI

### ISSUE THREAD

**Issue by michagruner** [docspell/android-client#40]
I have set up a docspell instance at home which is working fine so far. On that instance I configured a source to upload data via my android device. I've got two devices: a OnePlus 6 with Android 11 and a Redmi Note 9 Pro with MIUI 12.5.1. Both devices are configured to upload to that one share. It is working perfectly fine on the Oneplus and not at all on the Redmi. I wonder if there is a way to debug this. On the Redmi I already removed the app and the data and installed it completely new.

Any help is appreciated. Thanks, Michael

**Comment by eikek** [comment]
Hi @michagruner sad to hear this. I'm not sure how to debug this right now, I have to look into it. (If anyone with good (enough) android knowledge is reading this, I'm happy for any help here). I have a Redmi device, too with MIUI 11.0.8 (Android 9), where the app is working fine. What are you seeing in particular? Is the app not even starting? Or can you describe the symptoms?

**Comment by michagruner** [comment]
Hi @eikek the app is starting but then avoiding the upload. It simply does nothing. I noticed a similar behaviour when trying to upload a picture to twitter. The device let me choose the file to upload but then does nothing more. I did some more research on the upload thing and it seems to be the Redmi Optimisations (in the phones developer-setting) which I decided to switch of for privacy reasons. Switching it on again solved my problem with the docspell client. Very odd behaviour by MIUI from my perspective...thanks for your help again, hope this helps someone else experiencing similar things.
